Lib/multiprocessing/managers.py

```diff
@@ -1148,7 +1148,7 @@ def set(self, value):
 BaseListProxy = MakeProxyType('BaseListProxy', (
     '__add__', '__contains__', '__delitem__', '__getitem__', '__len__',
     '__mul__', '__reversed__', '__rmul__', '__setitem__',
-    'append', 'count', 'extend', 'index', 'insert', 'pop', 'remove',
+    'append', 'clear', 'count', 'extend', 'index', 'insert', 'pop', 'remove',
     'reverse', 'sort', '__imul__'
```

Comment:
Copy added to list at same time as clear.  See issue.
```suggestion
    'append', 'clear', 'copy', 'count', 'extend', 'index', 'insert', 'pop',
    'remove', 'reverse', 'sort', '__imul__'
```

Reply:
Added ListProxy.copy as you suggested